### DISCUSSION TITLE
Suspicion of wintrade

### DISCUSSION THREAD

**Discussion by noscripted** [chromiecraft/chromiecraft#6275] [General]

**Comment by BeefSteaker** [comment]
Hi, make a support ticket on the discord and we can look into it.

**Reply by noscripted** [reply]
I don't have a discord and I don't want one. What other information do you need for this? A good and experienced IT should be able to figure this out. Sttyler played for his life and Yaau just rested in the graveyard. Sttyler and Yaau were even together yesterday at the Scarlet Monastery and I don't think that was a coincidence.

**Comment by 55Honey** [comment]
@noscripted this is public. Everyone can read it and we do not condone public callouts whatsoever.
If you want to report something, please use private channels, like the support tickets on discord. We need to remove / sanction broken rules, public callouts the same as anything else.

**Reply by noscripted** [reply]
I do not insist that my post be public. It will be quite enough if you check it in complete anonymity. If you make your verification conditional on a ticket on discord, although I clearly wrote that I don't have discord, then you probably don't want to deal with it. Maybe *redacted* is some kind of VIP player. If you don't punish frauds/or deal with suspicions, it will definitely not pay off in the future, believe me. Any excessive correctness is ultimately an own goal.

**Reply by 55Honey** [reply]
> I do not insist that my post be public.

If you post it here, literally everyone can read it. It **is** public. No matter if you want it to be.
Again, public callouts - no matter who you name - are not allowed.

The highest Discord verification level is the only somewhat reliable way to protect our community from like scam links and fraud.
